ISISScientificComputing/autoreduce
# Pylint webapp reduction variables

Pylint changes to reduction variables in Webapp

To Test;
* Code review
* Start a local version of the WebApp and ensure that everything still renders correctly 

Fixes #129
